Source/CsDebugScript.Engine/Engine/Debuggers/DbgEngDll.cs

```diff
@@ -75,6 +75,23 @@ internal class DbgEngDll : IDebuggerEngine
         private static IDebugSystemObjects4 systemObjects;
 
         /// <summary>
+        /// Dictionary of all debugee flow controlers. Key is process id that is being debugged.
+        /// </summary>
+        private static DictionaryCache<uint, DebuggeeFlowController> debugeeFlowControlers;
+
+        /// <summary>
+        /// Static constructor.
+        /// </summary>
+        static DbgEngDll()
+        {
+            // Populate flow controlers lazely.
+            //
+            debugeeFlowControlers =
+                new DictionaryCache<uint, DebuggeeFlowController>(
+                    (processId) => new DebuggeeFlowController(originalClient));
```

Comment:
Added comment.

---
In reply to: [69979807](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69979807) [](ancestors = 69979807)